src/PKSim.Core/Model/Individual.cs

```diff
@@ -53,16 +53,7 @@ public Individual() : base(PKSimBuildingBlockType.Individual)
 
       public virtual bool IsHuman => Species.IsHuman;
 
-      public virtual bool IsPreterm
-      {
-         get
-         {
-            if (!Population.IsPreterm)
-               return false;
-
-            return OriginData.GestationalAge.HasValue && OriginData.GestationalAge.Value <= CoreConstants.NOT_PRETERM_GESTATIONAL_AGE_IN_WEEKS;
```

Comment:
@Yuri05 This is interesting here. Look at the constraint. We have 
.GestationalAge.Value **<=** CoreConstants.NOT_PRETERM_GESTATIONAL_AGE_IN_WEEKS;

But in another place in the code, we use **<** (string inferior) and I am not sure why.

Reply:
No idea. 
But actually there is a wish since a long time to restrict preterms creation not by GA<=40 , but by GA<=37 (or 38)

Reply:
ok. Can we create another issue for this?

Reply:
Think we have an issue for this already.. but maybe not. Will check.